### PR TITLE
feat(starfish): add tpm graph for top 5 tpm queries in db query details

### PR DESCRIPTION
**Pull request by DominikB2014**
Show a throughput per transaction graph, that displays the throughput for the top 5 highest throughput transaction in the db query details page

### CODE REVIEW

File: static/app/views/starfish/modules/databaseModule/queries.js [hunk 1]
```diff
@@ -87,6 +95,46 @@ export const getTopOperationsChart = (date_filters, interval) => {
   `;
 };
 
+export const useQueryTransactionByTPM = row => {
+  const pageFilter = usePageFilters();
+  const {startTime, endTime} = getDateFilters(pageFilter);
+  const dateFilters = `
+    greater(start_timestamp, fromUnixTimestamp(${startTime.unix()})) and
+    less(start_timestamp, fromUnixTimestamp(${endTime.unix()}))
+  `;
```
Comment (wmak) [inline static/app/views/starfish/modules/databaseModule/queries.js:104]: nit: We already create the date filters in `QueryDetailBody`

**Review by wmak — APPROVED** [review]
small nits, otherwise lgtm

File: static/app/views/starfish/modules/databaseModule/queries.js [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
+
 const DEFAULT_WHERE = `
   startsWith(span_operation, 'db') and
   span_operation != 'db.redis' and
   module = 'db' and
   action != ''
 `;
 
+const INTERVAL = 12;
+
```
[the remark this answers is not in the document]
Comment (DominikB2014) [reply static/app/views/starfish/modules/databaseModule/queries.js:15]: Yeah we can do that!

File: static/app/views/starfish/modules/databaseModule/queries.js (hunk 1 above)
Comment (DominikB2014) [reply static/app/views/starfish/modules/databaseModule/queries.js:104]: That's fair, but fwiw I do like the idea of using the page filters hook here, it makes this more "independant", you won't have pass in the page filters every time, cleans it up a bit.

**Pull request merged**